tests/test_examples/test_http_client_connection_to_aries_cloud_agent.py

```diff
@@ -0,0 +1,267 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
```

Comment:
The tests should be in `tests/test_packages/test_connections/test_http_client/*`

Reply:
Done